lib/helpers/initialize_adapter.js

```diff
@@ -7,6 +7,15 @@ module.exports = async function initializeAdapter(adapter = MemoryAdapter) {
   if (adapter === MemoryAdapter) {
     attention.warn('a quick start development-only MemoryAdapter is used');
   }
+  function isCtor() {
```

Comment:
creating a function like this is not necessary, just use the expressions in the if.